src/components/projects/projects-table/projects-table/projects-table.module.css

```diff
@@ -4,5 +4,4 @@
  */
 
 .table {
```

Comment:
I don't think you should leave empty CSS definitions. Having a CSS file which contains nothing other than the license comment at the top is totally fine